tasks/xpack/security/elasticsearch-security-native.yml

```diff
@@ -16,12 +16,30 @@
   set_fact: manage_native_roles=true
```

Comment:
Commenting here since I can't do it above. The vim swap file `tasks/xpack/security/.elasticsearch-security-native.yml.swp` is checked in.

Reply:
Not sure if this means I should stop working so early/late when I'm not fully awake, or give in and not switch to vim 😄 either way, removed in https://github.com/elastic/ansible-elasticsearch/pull/479/commits/9b8aab8a4e1f6253ba36a4dcb3a48b8f96b339d2